src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -99,236 +99,236 @@
-    <sys:String x:Key="Activate_Uninstall_All_Licenses">Uninstall all licenses</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Uninstall all installed licenses. You will need to install license after this operation.</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Licenses_Button">Uninstall</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses">Disinstalla tutte le licenze</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses.Description">Disinstalla tutte le licenze installate. Sarà necessario installare una licenza dopo questa operazione.</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Licenses_Button">Disinstalla</sys:String>
 
-    <sys:String x:Key="Activate_Keys_Management">Keys management</sys:String>
-    <sys:String x:Key="Activate_Keys_Management.Description">Install, uninstall keys or activate Office offline.</sys:String>
+    <sys:String x:Key="Activate_Keys_Management">Gestione delle chiavi</sys:String>
+    <sys:String x:Key="Activate_Keys_Management.Description">Installa, disinstalla le chiavi o attiva Office offline.</sys:String>
     <sys:String x:Key="Activate_Product_Key">Product key</sys:String>
-    <sys:String x:Key="Activate_Product_Key.Description">Install Office product key, product key format: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
-    <sys:String x:Key="Activate_Install_Product_Key">_Install key</sys:String>
-    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Uninstall all unactivated keys</sys:String>
-    <sys:String x:Key="Activate_Uninstall_All_Product_Keys">Uninstall _all keys</sys:String>
-    <sys:String x:Key="Activate_Install_CID">Install confirmation ID</sys:String>
-    <sys:String x:Key="Activate_Install_CID.Description">Install confirmation ID to activate Office offline.</sys:String>
-    <sys:String x:Key="Activate_Input_CID">Please enter the confirmation ID</sys:String>
+    <sys:String x:Key="Activate_Product_Key.Description">Installa product key di Office, formato della product key: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
+    <sys:String x:Key="Activate_Install_Product_Key">_Installa chiave</sys:String>
+    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Disinstalla tutte le chiavi non attivate</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Product_Keys">Disinstalla _tutte le chiavi</sys:String>
+    <sys:String x:Key="Activate_Install_CID">Installa confirmation ID</sys:String>
+    <sys:String x:Key="Activate_Install_CID.Description">Installa confirmation ID per attivare Office offline.</sys:String>
+    <sys:String x:Key="Activate_Input_CID">Inserire il confirmation ID</sys:String>
 
     <sys:String x:Key="Activate_KMS_Management">Gestione KMS</sys:String>
-    <sys:String x:Key="Activate_KMS_Management.Description">Set or reset the KMS host or port.</sys:String>
-    <sys:String x:Key="Activate_KMS_Apply">_Set host</sys:String>
```

Comment:
The hotkey `_I` is conflict with hotkey `_Installa chiave` in line 291. It's recommend to change it to `Impo_sta host`.